nfs-ganesha/build/build_deb

```diff
@@ -85,6 +85,15 @@ cd $WORKSPACE/nfs-ganesha
 # add debian directory next to src
 mv $WORKSPACE/nfs-ganesha-debian/debian $WORKSPACE/nfs-ganesha/
 
+# create and apply a patch to disable LizardFS FSAL
+echo "LS0tIGEvZGViaWFuL3J1bGVzCisrKyBiL2RlYmlhbi9ydWxlcwpAQCAtMjgsNiArMjgsNyBAQCBv
+dmVycmlkZV9kaF9hdXRvX2NvbmZpZ3VyZToKIAkJLURVU0VfRlNBTF9YRlM9T0ZGIFwKIAkJLURV
+U0VfRlNBTF9DRVBIPU9OIFwKIAkJLURVU0VfRlNBTF9SR1c9T04gXAorCQktRFVTRV9GU0FMX0xJ
+WkFSREZTPU9GRiBcCiAJCS1EVVNFX0ZTQUxfR1BGUz1PRkYgXAogCQktRFVTRV9GU0FMX1BBTkZT
+PU9GRiBcCiAJCS1EVVNFX0ZTQUxfR0xVU1RFUj1PRkYgXAo="|base64 -d > disable-lizardfs-fsal.patch
+
+patch -p1 < disable-lizardfs-fsal.patch
+
```

Comment:
Not a huge fan of embedding patches like this, but fine with removing LizardFS FSAL.